scripts/build_server_dev

```diff
@@ -4,10 +4,32 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
+CONFIG='clang'
+
+while getopts "at" opt; do
+    case "$opt" in
+    a)
+        CONFIG="asan"
+        bazel_build_flags+=(
+          "--compilation_mode=dbg"
```

Comment:
should we add this as part of the configs instead? if you specify asan you should build dbg by default

Reply:
Done.